project/FrontendBuild.scala

```diff
@@ -20,8 +20,12 @@ private object AppDependencies {
 
   def apply() = Seq(
     ws,
-    "uk.gov.hmrc"                 %% "frontend-bootstrap"       % "8.19.0",
+    "uk.gov.hmrc"                 %% "bootstrap-play-25"        % "1.5.0",
```

Comment:
Could we use bootstrap-play-25 version 1.6.0? There was a recent announcement 2 weeks ago that we only have 4 weeks left to update to the latest version.